public/stylesheets/social-media-buttons.css

```diff
@@ -0,0 +1,144 @@
+.resp-sharing-button__link,
```

Comment:
Interesting that you used `BEM` class naming convention, how did you think of it? Class naming convention like variables should be consistent, keep that in mind in the future.

Reply:
I did find these media share buttons as a sample on another site, i copied it and changed to fit the page, didn't touch the original naming. Will change it for consistency's sake.